auth-api/migrations/versions/2024_05_09_fcede9989aa2_.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
+
+Revision ID: fcede9989aa2
+Revises: a8c61064e6fd, 17369fa9416d
+Create Date: 2024-05-09 10:57:29.187758
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'fcede9989aa2'
+down_revision = ('a8c61064e6fd', '17369fa9416d')
```

Comment:
fix migration error:
Error: Multiple head revisions are present for given argument 'head'; please specify a specific target revision, '<branchname>@head' to narrow to a specific head, or 'heads' for all heads

Reply:
it's due to merge after migration, already removed.